config/routes.rb

```diff
@@ -86,25 +86,32 @@
     end
     resources :tokens, only: %i(index)
     resources :channels, constraints: { channel_id: %r{[^\/]+} }
-    namespace :stats do
-      get :signups_per_day
-      get :email_verified_signups_per_day
-      get :youtube_channels_by_view_count
-      get :twitch_channels_by_view_count
-      get :javascript_enabled_usage
-    end
+
+    # /api/v1/
     namespace :v1, defaults: { format: :json } do
+      # /api/v1/stats/
+      namespace :stats, defaults: { format: :json } do
+        get "/channels", to: "channels#channels"
+        get "/channels/:uuid", to: "channels#channels"
```

Comment:
I don't really like this design. We should follow a more isolated approach, where this actually routes to a different method in the controller.

This could be more rails-esque. If you name your function `index` and another one `show` in the channels controller, you can simply have this be 
`resource :channels, only: [:index, :show]`.

Reply:
There's a high chance we'll rename the routes & controllers once more statistics are added, 
I will leave this as-is for now, but I agree with your sentiment for a cleaner route

Reply:
Could we still split out the specific UUID lookup into a separate method?

Reply:
I agree with @CoryMcDonald. `ChannelsController` should follow a RESTful convention

Reply:
Fixed